packages/sst/src/constructs/StaticSite.ts

```diff
@@ -525,15 +525,6 @@ interface ImportMeta {
   private buildApp() {
     const { path: sitePath, buildCommand } = this.props;
 
-    // validate site path exists
-    if (!fs.existsSync(sitePath)) {
-      throw new Error(
-        `No path found at "${path.resolve(sitePath)}" for the "${
-          this.node.id
-        }" StaticSite.`
-      );
-    }
-
```

Comment:
hey @suil this is checking the path to the site, not the output generated by the build command.